modules/events.js

```diff
@@ -0,0 +1,29 @@
+import {
+  section1,
+  section2,
+  section3,
+  logo,
+  separator,
+} from './variables.js';
+
+export function handleListBookClick() {
+  section1.style.display = '';
+  section2.style.display = 'none';
+  section3.style.display = 'none';
+  separator.style.display = 'none';
+  logo.innerHTML = 'Awesome Books';
+}
+
+export function handleNewBookClick() {
+  section1.style.display = 'none';
+  section2.style.display = '';
+  section3.style.display = 'none';
+  logo.innerHTML = 'Awesome Books';
+}
+
+export function handleContactClick() {
+  section1.style.display = 'none';
+  section2.style.display = 'none';
+  section3.style.display = '';
+  logo.innerHTML = 'Awesome Books';
+}
```

Comment:
Please kindly note that as part of the requirement, you are to refactor your functions to arrow functions to follow the ES6 syntax 😄